geode-for-redis/src/main/java/org/apache/geode/redis/internal/data/delta/ZAddsDeltaInfo.java

```diff
@@ -24,6 +24,7 @@
 import java.util.List;
 
 import org.apache.geode.DataSerializer;
+import org.apache.geode.internal.InternalDataSerializer;
 
 public class ZAddsDeltaInfo implements DeltaInfo {
```

Comment:
Could this be renamed to `ZAddDeltaInfo` and the enum associated with it to `ZADD`?